testdata/ClientHello-JSON-Chrome102.json

```diff
@@ -0,0 +1,75 @@
+{
+	"cipher_suites": [
+        {"id": 2570, "description": "GREASE"},
```

Comment:
Not really. In this version we are only using `id` fields. And the `description` is only for human-readable purpose. 

Otherwise we might need to manually map every name into a value -- with very low marginal payback. Possibly we could find a better way to mass-generate this relation, with a crawler for RFC/IANA documents somehow?